arbiter-core/src/environment.rs

```diff
@@ -35,7 +35,7 @@ pub struct Environment {
     pub(crate) state: Arc<AtomicState>,
     pub(crate) evm: EVM<CacheDB<EmptyDB>>,
     pub(crate) socket: Socket,
-    pub seeded_poisson: SeededPoisson,
+    pub seeded_poisson: Option<SeededPoisson>,
```

Comment:
Whoa wait, why are we doing this in this PR? Also this is a pretty hacky solution. It does work (and I did propose it lol) but can we get this out of here? It doesn't fit the intention of the PR.